<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Splice</class>
 <widget class="QWidget" name="Splice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Splice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 12pt &quot;Arial&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Data root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DataRootLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="LoadData">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Infer root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="InferRootLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="LoadInfer">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Brain root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="BrainRootLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="LoadBrain">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Save root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SaveRootLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="Selet">
         <property name="text">
          <string>Selet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>561nm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox561"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>488nm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox488"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton561">
         <property name="text">
          <string>561nm</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton488">
         <property name="text">
          <string>488nm</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="SplicePrintText"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
